firstDirectory/topic3/Operations.java

```diff
@@ -0,0 +1,95 @@
+package firstDirectory.topic3;
+
+public class Operations {
+    static void printArithmeticOperations() {
+        System.out.println("ArithmeticOperations");
+        int a = 10;
+        int b = 110;
+        int sum = a+b;
+        int min = b-a;
+        int der = a*b;
+        double quotient = b/a;
```

Comment:
please consider using the % operator (restul impartirii) with int number

Reply:
> please consider using the % operator (restul impartirii) with int number

Thank you! Added b%a arithmeticl operation.